app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
OK so I confirmed that doorkeeper was made to work with user authentication. I keep getting an error as here: http://expertland.net/question/1mp6m4u8b6c4t72yuuo2mp3z52ob8l6/detail.html

SO it looks like the id error is the user id. 

Unless you have a better idea, I think we are going to have to set up a 'payroll' user to consume the data.